spec/fixtures/attachments/test-pattern.html

```diff
@@ -0,0 +1,235 @@
+<!DOCTYPE html>
```

Comment:
i downloaded our FAQ page because i was in a hurry to get things to work but there's probably a better fixture to put here - i really just need the .html extension